woodwork/table_accessor.py

```diff
@@ -347,6 +350,13 @@ def time_index(self):
             _raise_init_error()
         return self._schema.time_index
 
+    @property
+    def use_standard_tags(self):
+        """Whether standard tags are used for each column in the table"""
```

Comment:
When I first read this my thought was that this was returning a boolean indicating whether or not all columns had the same value. What if we said this instead? "A dictionary containing the `use_standard_tags` setting for each column in the table"

Reply:
Good point! Changing